zabbix/tests/test_zabbix.py

```diff
@@ -493,3 +493,31 @@ def test_zabbix_respect_true_ssl_verify(self):
 
     def test_zabbix_respect_default_ssl_verify(self):
         self.validate_requests_ssl_verify_setting(self.instance, True)
+
+    def test_zabbix_disabled_triggers_(self):
+        """
+        When there are no triggers enabled and return empty list for `trigger.get` call, then we are expecting no
+        problems and host will go back to green if it was red because there will be no event with problems.
+        """
+        # TODO this is needed because the aggregator retains events data across tests
+        aggregator.reset()
+
+        def _mocked_method_request(url, name, auth=None, params={}, request_id=1):
+            if name == "apiinfo.version":
+                return self._apiinfo_response()
+            elif name == "host.get":
+                return self._zabbix_host_response()
+            elif name == "problem.get":
+                response = self._zabbix_problem()
+                return response
+            elif name == "trigger.get":
+                return None
```

Comment:
Does this work properly? Given that the code is:

```
response = self.method_request(url, "trigger.get", auth=auth, params=params)
trigger = response.get('result')
```

I'm expecting the response to be `response['result'] = []`

Reply:
@whazenberg  correct I was wrong in this. I have made the change and also there should be an event for a host without any problems/triggers which will make the host healthy.